src/tests/plans/D3MOperatorPlan.t.sol

```diff
@@ -0,0 +1,108 @@
+// SPDX-FileCopyrightText: © 2022 Dai Foundation <www.daifoundation.org>
+// SPDX-License-Identifier: AGPL-3.0-or-later
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+pragma solidity ^0.8.14;
+
+import { D3MPlanBaseTest } from "./D3MPlanBase.t.sol";
+import { D3MOperatorPlan } from "../../plans/D3MOperatorPlan.sol";
+
+contract D3MOperatorPlanTest is D3MPlanBaseTest {
+
+    bytes32 ilk = "DIRECT-PROTOCOL-A";
```

Comment:
I don't think this is used anywhere.

Reply:
Btw feel free to leave any non important comment on the tests for a later stage.

Reply:
removed